assets/src/components/conversations/ConversationsProvider.tsx

```diff
@@ -16,20 +16,55 @@ import {
 } from '../../presence';
 import ConversationNotificationManager from './ConversationNotificationManager';
 
+type Inbox = {
+  conversationIds: string[];
+  unreadCount: number;
+};
+
+type Inboxes = {
```

Comment:
This structure will make it easier for us to expand to adding custom inboxes as well. 

* `all` represents all of the inboxes we support today.
* `bySource` will contain all of the inboxes that are channel specific.
* We can expand by adding something like `custom` which will contain all of the inboxes that the user configured themselves.